internal/svcmanager/svcmanager.go

```diff
@@ -72,23 +84,22 @@ func (m *Manager) Wait() error {
 	return m.WaitWithContext(context.Background())
 }
 
-func (m *Manager) Ready() bool {
+func (m *Manager) Ready() error {
 	if m.ready {
-		return true
+		return nil
 	}
 
 	if m.cfg.GetInt(constants.SvcConfigPort) == 0 {
-		return false
+		return errs.New("Service port not set in config")
```

Comment:
This isn't an error. It not being said simply means it's not ready because it hasn't started.